src/Symfony/Component/Validator/Constraints/LocaleValidator.php

```diff
@@ -42,13 +42,17 @@ public function validate($value, Constraint $constraint)
             throw new UnexpectedValueException($value, 'string');
         }
 
+        if (!class_exists(Locales::class)) {
+            throw new LogicException('The "symfony/intl" component is required to use the Locale constraint.');
```

Comment:
> The Intl component is required to use the Locale constraint. Try running "composer require symfony/intl".

Reply:
i copied it from other intl validators, tend to prefer the consistency :/

https://github.com/symfony/symfony/blob/b82b09eefb084e487997f4af753400d721edd0a8/src/Symfony/Component/Validator/Constraints/CountryValidator.php#L46

Reply:
Consistency with the other components is important as well. Here is my PR on that: #31392